stack.yaml

```diff
@@ -1,5 +1,4 @@
-resolver:
-  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/17/5.yaml
+resolver: lts-18.2
```

Comment:
:+1:

Mentioned elsewhere, but we want our custom snapshot to be how our _apps_ get stable dependencies. We don't want to be circularly inter-depending on it throughout our self-supplied dependencies. :recycle: